From 4fef44941d296ca7c807da39582a646f935fadfa Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Thu, 8 Dec 2016 13:09:04 +0000
Subject: [PATCH 1/2] Add $(CXX1XSTD) to PKG_CPPFLAGS

This causes Rcpp build failures that would otherwise not happen until
building a package that uses Rcpp and that does have this in src/Makevars,
for example readxl. A subsequent commit will fix the build failures that
this will introduce on macOS 10.7 which supports some C++11 features via
tr1. Full support of C++11 started with macOS 10.9 where libc++ is used
instead of an old version of libstdc++
---
 src/Makevars | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makevars b/src/Makevars
index f75dd3a..6543d0b 100644
--- a/src/Makevars
+++ b/src/Makevars
@@ -1,2 +1,2 @@
-PKG_CPPFLAGS =	-I../inst/include/
+PKG_CPPFLAGS =	-I../inst/include/ $(CXX1XSTD)
 
-- 
2.9.3

